greenkeeper.json

```diff
@@ -18,6 +18,7 @@
         "packages/cli/package.json",
         "packages/context/package.json",
         "packages/core/package.json",
+        "packages/eslint-config/package.json",
```

Comment:
+1 for updating the Greenkeeper config.

I think you did not follow all steps in our [Adding a new package checklis](https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md#adding-a-new-package), especially the following steps in [Register the new package](https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md#register-the-new-package):
 - Update MONOREPO.md - insert a new table row to describe the new package, please keep the rows sorted by package name.
 - Update docs/apidocs.html - add a link to API docs for this new package. (Maybe not needed?)
 - Update CODEOWNERS - add a new entry listing the primary maintainers (owners) of the new package.

Reply:
I was aware of that but did see `tslint-config` being registered. I guess it was not intentional.